extensions/wikia/CreateNewPageFlow/CreateNewPageFlow.hooks.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class CreateNewPageFlowHooks {
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$out, $skin". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)